QModManager/Patcher.cs

```diff
@@ -24,20 +25,22 @@ internal static void Patch()
             {
                 if (patched)
                 {
-                    Console.WriteLine("\nQMOD WARN: Patch method was called multiple times!");
+                    Console.WriteLine(LanguageLines.Patcher.CalledMultipleTimes);
                     return;
                 }
                 patched = true;
 
                 Hooks.Patch();
                 StartLoadingMods();
-                Hooks.Update += ShowErroredMods;
+                Hooks.Update += ShowDialog;
 
-                Hooks.OnLoadEnd();
+                VersionCheck.Check();
```

Comment:
Because this is now reaching out over the internet from a user's computer, we should respect the user's decision as to whether or not they wish to allow QMM to do this.

The installer wizard need to have an option somewhere reading something like "Allow QModManager to check for updates?"

That setting will need to be stored and checked here so that this is only executed if the user has given QMM permission to do so.

Reply:
Added information about this in the installer, and a toggle options in the mods menu to disable it.